core/conn/jdbc_type2/native/CSrvrConnect.cpp

```diff
@@ -268,10 +268,10 @@ void SRVR_CONNECT_HDL::addSrvrStmt(SRVR_STMT_HDL *pSrvrStmt,BOOL internalStmt)
         mapOfInternalSrvrStmt[pSrvrStmt->stmtName]= pSrvrStmt;  // +++ map error
     }
 
-
     //End of Soln. No.: 10-100202-7923
 
     count++;
+    //pSrvrStmt->myKey = count;
```

Comment:
this key will be needed once we use byte buffer to transport between JAVA and native, and it might be used for mapping the statements. I will remove the MFC related code one by one as in this work